### PR TITLE
fix FormTest

### PR DESCRIPTION
**Pull request by Balcan**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt [hunk 1]
```diff
@@ -174,11 +181,13 @@ class FormTest : BaseTest() {
             )
             pressBack()
             goToAnalytics()
-            waitToDebounce(3000)
+//            waitToDebounce(3000)
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt:184]: is this commented?, can we remove it?

File: app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt [hunk 2]
```diff
@@ -189,19 +198,13 @@ class FormTest : BaseTest() {
...
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt:201]: is this commented?, can we remove it?

File: app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt [hunk 3]
```diff
@@ -211,12 +214,14 @@ class FormTest : BaseTest() {
                 WARNING_COMPLETE_POSITION
             )
             scrollToBottomForm()
-            waitToDebounce(1000)
+//            waitToDebounce(1000)
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt:217]: is this commented?, can we remove it?

File: app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt [hunk 4]
```diff
@@ -226,91 +231,22 @@ class FormTest : BaseTest() {
                 ERROR_COMPLETE_POSITION
             )
             scrollToBottomForm()
-            waitToDebounce(1000)
+//            waitToDebounce(1000)
```
Comment (andresmr) [inline app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt:234]: is this commented?, can we remove it?

File: app/src/dhisUITesting/assets/databases/dhis_test.db
Comment (andresmr) [inline app/src/dhisUITesting/assets/databases/dhis_test.db:1]: did you updated the database?

**Review by andresmr — APPROVED** [review]

**Review by xavimolloy — APPROVED** [review]

**Pull request merged**